scripts/validate_map.py

```diff
@@ -187,6 +187,72 @@ def check_release_status_map_yes_sor_not(
     return map_not_sor
 
 
+def check_release_status_sor_yes_map_not(
+    df: pd.DataFrame, syn: Synapse, config: Dict, cohort: str, release: str
+) -> List:
+    """Check for codes where release status in scope of release is yes
+    but relase status in mapping file is not yes.
+
+    Args:
+        df (pd.DataFrame): dataframe representing map
+        syn (Synapse): Synapse object
+        config (dict): configuration parameters
+        cohort (str): cohort label to check
+        release (str): release label to check
+
+    Returns:
+        list: codes with lenient release status
+    """
+    map_status = df[cohort].str.lower()
+    map_type = df["data_type"].str.lower()
+    map_codes = df.loc[
+        ((map_status == "y") & ((map_type == "derived") | (map_type == "curated")))
```

Comment:
Why does map_type not map to map_status = y.

Reply:
these columns relate to different information for each variable listed in the mapping:

map_status: 
  def: whether the variable should be included in the cBioPortal files
  values: y, n 
map_type: 
  def: indicates category of the variable for etl
  values: derived, curated, code, etc.

since only derived and curated variables are listed in the scope of release, we only need to compare the variable release status between the mapping and scope of release for these variables